CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - SSL options for ``tt connect`` command.
 - An ability to pass arguments to a connect command.
 - ``tt binaries`` command. It shows a list of installed binaries and their versions.
+- A configurable variable `cluster_cookie` for `tt create cartridge` teamplate.
```

Comment:
No. This change is not included in version 1.0.0)

Reply:
I'm sorry, fixed.